pkgs/development/libraries/gaia/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libyaml
+, swig
+, pkgconfig
+, wafHook
+, makeWrapper
+, qt4
+, python
+, pythonSupport ? true
+# Default to false since it brakes the build
+, stlfacadeSupport ? false
+, assertsSupport ? true
+, cyclopsSupport ? true
+}:
+
+assert pythonSupport -> python != null;
+
+stdenv.mkDerivation rec {
+  pname = "gaia";
+  version = "2.4.5";
+
+  src = fetchFromGitHub {
+    owner = "MTG";
+    repo = "gaia";
+    rev = "v${version}";
+    sha256 = "12jxb354s2dblr2ghnl3w05m23jgzvrrgywfj8jaa32j3gw48fv2";
+  };
+
+  # Fix installation error when waf tries to put files in /etc/
+  prePatch = ''
```

Comment:
Superfluous empty string.

Reply:
Adding all of the superfluous empty strings/lists is kind of a habit of mine. The idea is, that if at some point in the future we'll add e.g a `prePatch` that will always be set, the diff is going to be much clearer.

Reply:
Well, I beg to differ, because while it avoids one line in the diff, it makes the code harder to read and I tend to prefer avoiding the latter, but anyway, it's just a matter of personal preference and as long we don't have a code style guide that explicitly bans this I'm fine.

Reply:
> as long we don't have a code style guide that explicitly bans this I'm fine.

I wonder whether `nix-fmt` will handle this when the day comes and it'll work in production..

I hope it's OK with you that I'll mark this as resolved although no change was done. I wouldn't mind that much changing it if another reviewer will say he doesn't like it.